backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/analysis/model/impl/ElasticsearchAnalysisDefinitionRegistry.java

```diff
@@ -110,4 +119,40 @@ public Map<String, CharFilterDefinition> getCharFilterDefinitions() {
 		return Collections.unmodifiableMap( charFilterDefinitions );
 	}
 
+	@Override
+	public Optional<ElasticsearchAnalyzerDescriptor> analyzerDescriptor(String name) {
+		if ( analyzerDefinitions.containsKey( name ) ) {
+			return Optional.of( new ElasticsearchAnalysisDescriptor( name ) );
+		}
+		else {
+			return Optional.ofNullable( ElasticsearchAnalysisDescriptor.BUILT_IN.get( name ) );
```

Comment:
I'm not sure it's a good idea to expose built-in analyzers in metadata when we have no idea what they look like on the server...

For metadata, it would probably be safer to only expose what is actually defined, not what *might* exist on the server?

Note it's not the same with Lucene, because with that backend we actually have a full view of which analyzers are defined.